.copier-answers.yml

```diff
@@ -18,5 +18,3 @@ pypi: false
 repo_name: sas-bluesky
 strict_typing: false
 type_checker: pyright
-
-#copier updated
```

Comment:
This usually is a sign that the copier update wasn't without problems [and looking at the PR, it failed to update from 3.0.0 -> 4.x](https://github.com/DiamondLightSource/sas-bluesky/pull/26), which is expected (4.0.0 made the change that allows it to automatically update, but only if it is updating from >4.0.0)